nulinks-common/test/__snapshots__/data.search.test.js.snap

```diff
@@ -32,7 +32,11 @@ exports[`searching data.js matches snapshot for term faf 1`] = `"FAFSA"`;
 
 exports[`searching data.js matches snapshot for term fas 1`] = `"FAFSA"`;
 
-exports[`searching data.js matches snapshot for term fin 1`] = `"My Financial Aid Status"`;
+exports[`searching data.js matches snapshot for term fin 1`] = `"My Schedule"`;
```

Comment:
This change is maybe not preferable. I think perhaps most people check their financial aid status more than their finals schedule, but it really depends on the part of the year. Regardless, a few more letters gets the user to the right place and they're both pretty infrequent so I think we're ok on this.